docs/index.html

```diff
@@ -2570,10 +2591,164 @@ <h3 class='fl m0' id='sendlog'>
       
         <div class='space-bottom0'>
           <div>
-            <span class='code bold'>metadata</span> <code class='quiet'>([<a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array">array</a>])</code> an array of extra metadata (will go through JSON.stringify)
+            <span class='code bold'>metadata</span> <code class='quiet'>([any])</code> extra metadata (will go through JSON.stringify).
+Can be passed as any number of trailing arguments.
+
+          </div>
+          
+        </div>
+      
+    </div>
+  
+
+  
+
+  
+    
+      <div class='py1 quiet mt1 prose-big'>Returns</div>
+      <code><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise">promise</a></code>:
+        a promise of the success of the operation
+
+      
+    
+  
+
+  
+
+  
+
+  
+
+  
+
+  
+</section>
+
+          
+        
```

Comment:
docs are autogenerated from the source comments using the `documentation` npm package